### DISCUSSION TITLE
How to control citation rendered output?

### DISCUSSION THREAD

**Discussion by lukaemon** [quarto-dev/quarto-cli#4238] [Q&A] [answered]
I use citaiton like this `@raeScalingLanguageModels2022` in .qmd.  
The rendered result is Rae et al. ([2022](https://blah/#ref-raeScalingLanguageModels2022)).  
How do I get rid of parentheses? My target output is Rae et al. [2022](https://blah/#ref-raeScalingLanguageModels2022). 

Is it about changing [citation style](https://quarto.org/docs/authoring/footnotes-and-citations.html#citation-style)? 
I tried [Chicago Manual of Style 17th edition (author-date, Français)](https://www.zotero.org/styles/chicago-author-date-fr) with `csl: asset/chicago-author-date-fr.csl` and the the render is the same as default. However, the citation example of that style is, Hisakata, Nishida, et Johnston 2016; Musk 2006; Hogue 2001; Sambrook et Russell 2001, no paren around the year. 

What did I miss?

**Comment by rgaiacs** [comment, marked as answer]
[Minimal working example](https://github.com/quarto-dev/quarto-cli/files/10657730/mwe.zip)

Dear @lukaemon. [Citation Style Language](https://citationstyles.org/) only supports "normal" citations (like `[@raeScalingLanguageModels2022]`). It does **not** support in-text citations (like `@raeScalingLanguageModels2022`).

If you render the provided minimal working example, you will see that

```
Citation: [@knuth_art_1997].

In-text citation: @knuth_art_1997.
```

is render as

```
Citation: (Knuth, Addison-Wesley, et Education 1997).

In-text citation: Knuth, Addison-Wesley, et Education (1997).
```

> How do I get rid of parentheses?

You will have to write your own filter to remove the parentheses.

**Reply by lukaemon** [reply]
Beautiful! Exactly what I need. Thank you so much.
